Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -609,7 +609,7 @@ public void onPlayerBucketEmpty(final PlayerBucketEmptyEvent event) {
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
-        final String cmd = event.getMessage().toLowerCase(Locale.ENGLISH).split(" ")[0].replace("/", "").toLowerCase(Locale.ENGLISH);
+        final String cmd = event.getMessage().split(" ")[0].replace("/", "").toLowerCase(Locale.ENGLISH);
```

Comment:
What does this fix?

Reply:
nothing but why lower case the string two times